src/ensembl/stories/shared-components/badged-button/BadgedButton.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+
+import Roundbutton from 'src/shared/round-button/RoundButton';
+import BadgedButton from 'src/shared/badged-button/BadgedButton';
+import { SecondaryButton } from 'src/shared/button/Button';
+import { ReactComponent as DownloadIcon } from 'static/img/track-panel/download.svg';
+import styles from './BadgedButton.stories.scss';
+import ImageButton, {
+  ImageButtonStatus
+} from 'src/shared/image-button/ImageButton';
+
+const onClick = action('button-click');
+
+storiesOf('Components|Shared Components/BadgedButton', module)
+  .add('badged Button', () => (
+    <div className={styles.wrapper}>
+      <BadgedButton badge={':)'}>
+        <SecondaryButton onClick={onClick}>Primary button</SecondaryButton>
```

Comment:
Just to avoid confusion, perhaps the text inside the secondary button should not say "Primary button"? 😄

Reply:
Haha 😄 
Yes, I've update it.